src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -133,12 +138,12 @@ public async Task<ImmutableArray<IntentProcessorResult>> ComputeIntentAsync(Docu
                         dialogAction.ViableMembers,
                         dialogAction.PickMembersOptions,
                         selectedAll: true);
-                    var operations = await dialogAction.GetOperationsAsync(originalSolution, options, cancellationToken).ConfigureAwait(false);
+                    var operations = await dialogAction.GetOperationsAsync(originalSolution, options, progressTracker, cancellationToken).ConfigureAwait(false);
```

Comment:
Use CodeAnalysisProgress.None here instead of passing it in through 2 local functions

Reply:
i actually prefer this as it means if we fix the root issue with intents that we could pass it along.  Basically, update all code so taht it *can* work with progress, and then push taht all the way up to the point where we see the lack of support making it not possible.  At that *high* point pass in the 'none' to make things work, with sadness that we had to do that.